userguide/storage.rst

```diff
@@ -209,7 +209,12 @@ the details when considering whether encryption is right for your
 
 * This is full disk encryption and **not** per-filesystem encryption.
   The underlying drives are first encrypted, then the pool is created
-  on top of the encrypted devices.
+  on top of the encrypted devices. As data is written, it is automatically 
+  encrypted, and as data is read, it is decrypted on the fly. 
+
+* Data in the ARC cache and the contents of RAM are **always unencrypted**, 
+  and Swap is **always encrypted** (even on unencrypted volumes). The 
+  LOG (ZIL) cache is **encrypted if it relates to an encrypted disk**.
```

Comment:
After technical review, here is what we came up with:

*Data in memory, including ARC, is not encrypted. ZFS data on disk, including ZIL and SLOG, are encrypted if the underlying disks are encrypted. Swap data on disk is always encrypted.*